test/FastTests/Voron/CompactTreeTests.cs

```diff
@@ -185,7 +185,55 @@ public void CanDeleteLargeNumberOfItemsInRandomOrder()
                 Assert.Equal(0, tree.State.NumberOfEntries);
                 Assert.Equal(0, tree.State.BranchPages);
                 Assert.Equal(1, tree.State.LeafPages);
-                Assert.Equal(1, tree.State.Depth);
+            }
+        }
+
+        [RavenTheory(RavenTestCategory.Voron)]
+        [InlineData(400000)]
```

Comment:
@redknightlois I see we have more tests playing with 400000 entries. I will move those cases to SlowTests

Reply:
No problem